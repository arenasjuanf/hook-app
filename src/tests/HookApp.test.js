import { HookApp } from '../HookApp';
import { shallow } from 'enzyme';

describe("<HookApp /> Tests", () =>{

    test("should display correctly", () => {
        const wrapper = shallow( <HookApp /> );
        expect(wrapper).toMatchSnapshot();
    });

});