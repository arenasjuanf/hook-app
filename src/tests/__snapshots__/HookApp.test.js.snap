// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HookApp /> Tests should display correctly 1`] = `
<div>
  <h1>
    Hello world!
  </h1>
</div>
`;
